.github/workflows/ci_tests.yml

```diff
@@ -62,11 +62,6 @@ jobs:
             python: '3.10'
             tox_env: 'pycodestyle'
 
-          - name: 'ubuntu-build_docs'
-            os: ubuntu-latest
-            python: '3.10'
-            tox_env: 'build_docs'
-
```

Comment:
Can you please revert this deletion? I've figured out what the issue is and will have a fix for this build merged shortly.

Reply:
Reverted.